index.html

```diff
@@ -0,0 +1,487 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <link rel="stylesheet" type="text/css" href="./Assets/stylesheet.css">
+        <title>Space Ripples Reveal Big Bang's Smoking Gun</title>
+    </head>
+    <body>
+            <div class="aside">
+                    <span><strong>Editor's pick</strong></span>
+                    <hr>
+                    <div>
+                        <img src="./Assets/31SCI-SATELLITES2-threeByTwoSmallAt2X.jpg" alt="A link to the past">
+                        <p>Anarchy in the U.K</p>
+                    </div>
+                    <div>
+                        <img src="./Assets/31SCI-SATELLITES2-threeByTwoSmallAt2X.jpg" alt="A link to the past">
+                        <p>Anarchy in the U.K</p>
+                    </div>
+                    <div>
+                        <img src="./Assets/31SCI-SATELLITES2-threeByTwoSmallAt2X.jpg" alt="A link to the past">
+                        <p>Anarchy in the U.K</p>
+                    </div>
+                    <div>
+                        <img src="./Assets/31SCI-SATELLITES2-threeByTwoSmallAt2X.jpg" alt="A link to the past">
+                        <p>Anarchy in the U.K</p>
+                    </div>
+                </div>
+                <div class="aside-lower">
+                    <h3><strong>Sign Up for Science Times</strong></h3>
+                    <p>We’ll bring you stories that capture the wonders of the human body, nature and the cosmos.</p>
+                    <button class="subscribe-btn">Sign Up</button>
+                </div>
+        <header>
+            <h1>
+                <span>Space Ripples Reveal</span> 
+                <span>Big Bang's Smoking Gun</span>
+            </h1>
```

Comment:
In this project leave put only the navbar inside the `<header>`. The Title of the article and the main cover must be with inside the main content of the page.